src/main/java/MainWindow.java

```diff
@@ -54,7 +54,12 @@ public void setDuke(Duke duke) {
     @FXML
     private void handleUserInput() {
         String input = userInput.getText();
-        String response = duke.getParser().parse(input);
+        String response;
+        try{
+            response = duke.getParser().parse(input);
+        }catch(DukeException e){
```

Comment:
The parentheses here should have a whitespace separator. This is a coding style issue, but one that can be easily fixed through automation. May I suggest using the in-built code style and its refactoring ability in IntelliJ? :)